frontend/components/GasTable.vue

```diff
@@ -44,11 +38,9 @@
                         <span> Max Fee Per Gas </span>
                     </td>
                     <td class="Body">
-                        <FormatCurrency
-                            v-if="gasParameters && gasParameters.maxFeePerGas"
-                            :value="parseInt(gasParameters.maxFeePerGas)"
```

Comment:
Please convert it to BigNumber, not number and let's still use `FormatCurrency` since it's animated (: